project.mml

```diff
@@ -1644,6 +1644,58 @@ Layer:
         ) AS power_towers
     properties:
       minzoom: 14
+  - id: power-poles
```

Comment:
That's ok to group all power points into one layer, but portal ways shouldn't be rendered the same as power lines. They do not transmit power and could be confused with actual lines.

Reply:
I agree they should be rendered differently. 

But the layer just determines when they render "on top of" or "under" other layers. You can still render the power=portal ways with a different width and color, and if you use ORDER BY as shown in my example you can have them render above `power=line/minor_line/cable` features, if desired (or if it's more appropriate they can render under all those features by changing the order)

The only difference is that they would render below the subsequent layers like placenames, stations, amenity-points, and power-tower. This is better, because line features should generally be below icons and text labels.

Reply:
Layers have been merged and it should be ok this way